sections/collapsible-content.liquid

```diff
@@ -0,0 +1,383 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content collapsible-{{ section.settings.layout | split: '_' | first }}-layout color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content__wrapper{% if section.settings.layout == 'section_container' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div class="collapsible-content__header" style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption | escape }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="collapsible-content__heading">{{ section.settings.heading | escape }}</h2>
+        {%- else -%}
+          <h2 class="visually-hidden">{{ 'accessibility.collapsible_content_title' | t }}</h2>
+        {% endif %}
+      </div>
+      <div class="grid grid--1-col grid--2-col-tablet collapsible-content__grid{% if section.settings.desktop_layout == 'text_first' %} collapsible-content__grid--reverse{% endif %}">
+        {%- if section.settings.image != blank -%}
+          <div class="grid__item grid__item--image">
```

Comment:
Since the styles applied to `.grid__item--image` are really only valid in the context of the collapsible-content section, we shouldn't use a modifier of `.grid__item`, but rather a selector that's associated with collapsible-content. Something like `collapsible-content__grid-item--image`

Reply:
I did `collapsible-content__grid-item` so I don't use a modifier otherwise I'd also need `collapsible-content__grid-item`